### DISCUSSION TITLE
Show Product Name when Choosing Modifiers

### DISCUSSION THREAD

**Discussion by rsidhaarth** [Blair2004/NexoPOS#446] [Ideas]
Hi Team,

I just went here

![image](https://user-images.githubusercontent.com/16856248/147818890-9a9c5300-2006-49d1-b93e-e655ba64717b.png)

Just typed 22, I have kept the same number of SKU and Bar Code, and it shows the POP UP, but it is missing product name

![image](https://user-images.githubusercontent.com/16856248/147818917-c3b334d4-656a-4cb5-a019-1064b2e9bb2f.png)

It will be better if you show the product name as well and so the cashier can verify which product he is adding to the cart.

**Comment by Blair2004** [comment]
Hi, 
nice once.
Will be merged.
